build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This script looks like it does what is in the readme. Why not just reference it from the readme?